Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
 
-    collectionCounters: [],
+    lolCounters: [],
 
     /**
      * This function must be called when an ajax call is done, to ensure
```

Comment:
- eslint: Expected dot to be on same line as object.
- eslint: Strings must use doublequote.